Translated/Quests/Story/09 Rescuing Bruder 2/qst_1200090030.csv

```diff
@@ -1,6 +1,6 @@
 name010#0,"""Manon"""
 name020#0,"""Aina"""
 name030#0,"""Bruder"""
-text010#0,"""いけるわよ！\u3000アイナ！"""
-text020#0,"""うん！"""
-text030#0,"""よし、一気に決めるぞ！"""
+text010#0,"""We can do this, Aina!"""
+text020#0,"""Yeah!"""
+text030#0,"""Alright, let's settle it at once!"""
```

Comment:
Can you come up with a more natural way to write this line?

Reply:
made less awkward